src/frontend/App/hooks/useHentKontantstøtteUtbetalinger.ts

```diff
@@ -0,0 +1,41 @@
+import { Ressurs, RessursStatus } from '../typer/ressurs';
+import { useApp } from '../context/AppContext';
+import { useEffect, useState } from 'react';
+import { AxiosRequestConfig } from 'axios';
+
+export enum EFinnesKontantstøtteUtbetaling {
```

Comment:
```suggestion
export enum FinnesKontantstøtteUtbetaling {
```

Ser det har vært praksis tidligere at man prefixer alle enums med E, men personlig synes jeg det både ser litt rart ut og er redundant. Du kan selv vurdere om du vil fjerne E prefixen, selvfølgelig

Reply:
Enig, liker best uten prefix selv. Fikset